packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -52,6 +52,11 @@ class PinCodeBloc extends $PinCodeBloc {
     required this.autoPromptBiometric,
   }) {
     authenticated.connect().addTo(_compositeSubscription);
+    Rx.merge([
```

Comment:
As mentioned the max length check should be applied on adding new digits

Reply:
Added